enforcer-parent/enforcer/src/main/resources/Dockerfile

```diff
@@ -14,11 +14,11 @@
 # limitations under the License.
 # -----------------------------------------------------------------------
 
-FROM adoptopenjdk/openjdk11:jre-11.0.14.1_1-alpine
+FROM adoptopenjdk/openjdk11:jre-11.0.16.1_1
```

Comment:
This is an ubuntu based image, right?
These two images have the same digest
- https://hub.docker.com/layers/adoptopenjdk/openjdk11/jre-11.0.16.1_1/images/sha256-a5c8b107a919e5ec44776890eff01d39a612ffebd43c4ee36abd2fd8fa72005e?context=explore
- https://hub.docker.com/layers/adoptopenjdk/openjdk11/jre-11.0.16.1_1-ubuntu/images/sha256-a5c8b107a919e5ec44776890eff01d39a612ffebd43c4ee36abd2fd8fa72005e?context=explore